lib/datastore/request.js

```diff
@@ -465,17 +460,13 @@ DatastoreRequest.prototype.delete = function(keys, callback) {
  *     stream instance is returned.
  *
  * @example
- * //-
- * // Where you see `transaction`, assume this is the context that's relevant to
- * // your use, whether that be a Dataset or a Transaction object.
- * //-
  *
  * // Retrieve 5 companies.
- * transaction.runQuery(query, function(err, entities, endCursor, apiResponse) {
+ * dataset.runQuery(query, function(err, entities, endCursor, apiResponse) {
```

Comment:
+1 - seems weird.. Are we properly scoping variables between examples? Maybe it's picking up a previously defined `query` variable?